### DISCUSSION TITLE
Sort hashtag videos by created_at attribute

### DISCUSSION THREAD

**Discussion by keshaviscool** [davidteather/TikTok-Api#921] [Q&A] [answered]
Hi,

Is it possible or some API is available where I can sort the hashtag videos?

**Comment by batmanscode** [comment, marked as answer]
@KeshavSharmaWeb you could load the data into a pandas dataframe and filter things that way

**Reply by keshaviscool** [reply]
The data is HUGE. For #cringe, there are about 9lac videos. And we will be having around 1000 hashtags.. if we try to filter like that, we will be in big trouble

**Reply by batmanscode** [reply]
It would definitely be better to have some sort of filtering before the scrape. For example twint has a `min_likes` parameter so it only scrapes tweets above that

It doesn't seem like this is possible with this library at the moment. For a large dataframe like that pandas can process things in chunks and alternatively you can try out dask and polars if you want to try out something faster

Personally, pandas has been enough for even the largest dataframes I've worked with which was ~800k rows of bioinformatics data

**Reply by davidteather** [reply]
+1 you shouldn't have a problem sorting that many
